src/lib/request/read-many.request.ts

```diff
@@ -144,12 +144,7 @@ export class CrudReadManyRequest<T> {
     toResponse(data: T[], total: number): PaginationResponse<T> {
         const take = this.findOptions.take;
         const dataLength = data.length;
-        const nextCursor = PaginationHelper.serialize(
-            _.pick(
-                data.at(-1),
-                this.primaryKeys.map(({ name }) => name),
-            ) as FindOptionsWhere<T>,
-        );
+        const nextCursor = PaginationHelper.serialize(_.pick(data.at(-1), this.paginationKeys) as FindOptionsWhere<T>);
```

Comment:
if using `order by` in search, 
I think the nextCursor should contain the `sorting keys` with the paginationKeys.

That would solve both of the problems (https://github.com/woowabros/nestjs-library-crud/issues/456#issuecomment-1947596297)

Reply:
Good! I would change nextCursor to reflect order keys.